antlr4ide.ui/META-INF/MANIFEST.MF

```diff
@@ -23,7 +23,8 @@ Require-Bundle: antlr4ide.core;visibility:=reexport,
  org.eclipse.core.resources,
  org.eclipse.xtend,
  org.eclipse.xpand,
- org.eclipse.xtext.xbase.lib
+ org.eclipse.xtext.xbase.lib,
+ org.eclipse.core.variables;bundle-version="[3.2.0,4.0.0)"

```

Comment:
I do not know for sure because I have copied the version range from here: https://github.com/eclipse/eclipse.platform.debug/blob/master/org.eclipse.debug.ui/META-INF/MANIFEST.MF#L77

According to the Javadoc API at least version 3.0 is needed.